### PR TITLE
Conditionally support msgpack on Android only for Lollipop and later

### PR DESCRIPTION
**Pull request by paddybyers**
Fixes: https://github.com/ably/ably-java/issues/261

@psolstice pls review if you're around

### CODE REVIEW

File: android/src/main/java/io/ably/lib/util/Platform.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package io.ably.lib.util;
+
+import android.os.Build;
+
+public class Platform {
+	/* The msgpack-core library at 0.8.11 has issues on 4.4.2 and probably earlier */
```
Comment (mattheworiordan) [inline android/src/main/java/io/ably/lib/util/Platform.java]: Should we not link to https://github.com/ably/ably-java/issues/261?

**Comment by mattheworiordan** [comment]
@paddybyers is this not merged into 0.9?

**Comment by paddybyers** [comment]
> is this not merged into 0.9?

no, I will rebase 0.9, or @psolstice will

**Comment by mattheworiordan** [comment]
> is this not merged into 0.9?

Sorry, I actually meant 0.8 i.e. why is this PR still open if https://github.com/ably/ably-java/commit/f59fdaac3e42022849d658c676d8c36e6889f382 is in master?

**Comment by paddybyers** [comment]
> Sorry, I actually meant 0.8 i.e. why is this PR still open if f59fdaa is in master?

I merged and pushed to `master` and expected Github to keep up, but it didn't - probably I rebased before merging.

**Pull request closed without merging**